source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -990,8 +990,10 @@ export class TextBlock extends CardElement {
         this.color = Utils.getEnumValueOrDefault(Enums.TextColor, json["color"], this.color);
         this.isSubtle = json["isSubtle"];
         this.wrap = json["wrap"] === undefined ? false : json["wrap"];
+        if (typeof (json["maxLines"]) === "number") {
```

Comment:
You seem to have an indentation problem, the body of the if block isn't indented properly.

Reply:
yeah my commit editor was acting weird. I also fixed to use the current `typeof` practice